core/tests/protractor/learnerFlow.js

```diff
@@ -47,10 +47,10 @@ describe('Learner dashboard functionality', function() {
   var learnerDashboardPage = null;
   var oppiaLogo = element(by.css('.protractor-test-oppia-main-logo'));
   var continueButton = element(by.css('.protractor-test-continue-button'));
-  var clickContinueButton = function() {
+  var clickContinueButton = async function() {
     waitFor.elementToBeClickable(
```

Comment:
Shouldn't there be an await here and in front of other waitFors?

Might want to do a grep for waitFor and check that every waitFor has an await before it. Maybe grepping for "  waitFor" (two spaces before waitFor) is a quick way to do it.